netutil/ip_ranges.go

```diff
@@ -0,0 +1,50 @@
+package netutil
+
+import (
+	"fmt"
+	"net"
+	"strings"
+)
+
+// Default private IP ranges in CIDR notation
+const DefaultPrivateIPRanges = "10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,127.0.0.0/8,169.254.0.0/16,fc00::/7,fe80::/10"
```

Comment:
Do we need to export this?

Reply:
Just let it be exported in case someone wants to use the default IP ranges. This need not be exported